bundles/binding/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoHubBinding.java

```diff
@@ -85,6 +85,11 @@ protected void execute() {
         try {
             // send info request
             final InfoResponse response = createProtocol().info();
+            if (response == null) {
+                logger.error("Failed to connect to neohub [host '{}' port '{}']",
+                        new Object[] { hostname, port });
```

Comment:
The `new Object[] {hostname, port } ` method signature isn't needed, as you can just pass variable arguments as `hostname, port`.